### DISCUSSION TITLE
请问mortal2代比1代强度到底怎么样呢？

### DISCUSSION THREAD

**Discussion by lewissue** [Equim-chan/Mortal#29] [Q&A]
我最近发现2代官网上显示是b75c256，层数比1代的b40c192多了许多，应该更强。
但是看文档里strength似乎打不过1代，这是为什么啊？这个AI是强化学习，不应该和所用数据集有太大关联吧。作者能不能解答下这个疑惑，谢谢！

**Comment by Equim-chan** [comment]
目前从统计数据来看 v2 和 v1 的强度是相当的。

层数这个问题，不能用一般 DL 任务的眼光来看待 Mortal 的任务，这里不是越深越好的（要是真有这种好事我也不至于搞了一年多）。线上的这个 v2 版本加深的目的主要是以实验为目的，但是后来感觉拖得比较久了就把这个实验版本先上了。

**Reply by lewissue** [reply]
明白了。看来不像围棋那种完全信息游戏，日麻还是太复杂>_<
作为麻将玩家很感谢m老师的帮助，奥利给！
